marbot.yml

```diff
@@ -1407,6 +1413,42 @@ Resources:
       Targets:
       - Arn: !Ref Topic
         Id: marbot
+  ECSSpotFailureEvent:
+    Condition: ECSSpotNotificationsEnabled
+    DependsOn: TopicEndpointSubscription
+    Type: 'AWS::Events::Rule'
+    Properties:
+      Description: 'ECS Spot failure. (created by marbot)'
+      EventPattern:
+        source:
+        - 'aws.ecs'
+        'detail-type':
+        - 'ECS Service Action'
+        detail:
+          eventName:
+          - SERVICE_TASK_PLACEMENT_FAILURE
+      State: ENABLED
+      Targets:
+      - Arn: !Ref Topic
+        Id: marbot
+  ECSSpotInterruptionEvent:
+    Condition: ECSSpotNotificationsEnabled
+    DependsOn: TopicEndpointSubscription
+    Type: 'AWS::Events::Rule'
+    Properties:
+      Description: 'ECS Spot interruption notice. (created by marbot)'
```

Comment:
ECs task is shut down due to Fargate Spot interruption. (created by marbot)

Reply:
✅